src/main/java/samples/authentication/standAloneJwt/JwtCredentials.java

```diff
@@ -0,0 +1,416 @@
+package samples.authentication.standAloneJwt;
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JOSEObject;
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.JWSObject;
+import com.nimbusds.jose.Payload;
+import com.nimbusds.jose.crypto.RSASSASigner;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.security.KeyStore;
+import java.security.KeyStore.PrivateKeyEntry;
+import java.security.cert.CertificateEncodingException;
+import java.security.cert.X509Certificate;
+import java.security.MessageDigest;
+import java.security.PrivateKey;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.KeyStore.PasswordProtection;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.StringTokenizer;
+
+/* <summary>
+ * This class is not expected to run on Linux VMs on Travis CI.
+ * This sample will not pass due to issues with JWT and keystores on Linux machines.
+ * </summary>
+ */
+public class JwtCredentials {
```

Comment:
Why is this filename/class-name JwtCredentials ? Shouldn't it be StandAloneJWT ? Similar to csharp

Reply:
Folder structure has been refactored.